.github/workflows/ci.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       - name: Setup env
         run: |
           echo "NEMU_HOME=$GITHUB_WORKSPACE" >> $GITHUB_ENV
+          git checkout master
```

Comment:
no need for this. CI will automatically checkout to the HEAD commit to run tests

Reply:
ok